tests/phpunit/includes/Core/Storage/FakeHas_Connected_Admins.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * Class Google\Site_Kit\Tests\Core\Storage\FakeSetting
+ *
+ * @package   Google\Site_Kit\Tests\Core\Storage
+ * @copyright 2019 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Tests\Core\Storage;
+
+use Google\Site_Kit\Core\Storage\Has_Connected_Admins;
+
+class FakeHas_Connected_Admins extends Has_Connected_Admins {
```

Comment:
IMO the most important thing to test is how the Has_Connected_Admins behaves when it receives information from the database. How it receives/queries that information is less important since it uses core functionality to pull it. Reworking it to use real users from the database will only increase complexity of the test case and will slow it down. Not sure if we really need to go down the rabbit hole.

Reply:
> Reworking it to use real users from the database will only increase complexity of the test case and will slow it down.

It will slow the test down to some extent, but _not only_ that. It will provide more confidence by covering the whole functionality and will also be more clear as to what the class actually does and how it works.

Reply:
Ok, removed this fake class.